### PR TITLE
Enable mprintf and fmprint in scp and sftp (https://github.com/PowerShell/Win32-OpenSSH/issues/534)

### PR DESCRIPTION
**Pull request by bingbing8**
printf still does not work well so still keep the overriding until find the cause.

### CODE REVIEW

File: utf8.c [hunk 1]
```diff
@@ -259,11 +262,56 @@ vfmprintf(FILE *stream, const char *fmt, va_list ap)
 {
 	char	*str;
 	int	 ret;
-
+		
 	if ((ret = vasnmprintf(&str, INT_MAX, NULL, fmt, ap)) < 0)
 		return -1;
+#ifdef WINDOWS
```
Comment (bagajjal) [inline utf8.c:268]: Please move this functionality to posix layer and reduce the use of #ifdef windows..

File: openbsd-compat/openssl-compat.c [hunk 2]
```diff
@@ -66,6 +69,87 @@ ssh_compatible_openssl(long headerver, long libver)
 	return 0;
 }
 
+int
+get_wcwidth(wchar_t wc) {
+	if (0x20 <= wc && wc <= 0x7e)
```
Comment (bagajjal) [inline openbsd-compat/openssl-compat.c:74]: can you please add some comments, what this method is doing and how to interpret the hex codes (may be a URL that has these values will help us)..

**Review by bagajjal — CHANGES_REQUESTED** [review]

File: openbsd-compat/openssl-compat.c (hunk 2 above)
Comment (bingbing8) [reply openbsd-compat/openssl-compat.c:74]: for each range, I have comments near it. like /* Hiragana */. Is it okay?

**Comment by manojampalam** [comment]
There were new changes to utf8.c implementation in Windows that makes these set of changes obsolete.

**Pull request closed without merging**